code/datums/chat_message.dm

```diff
@@ -103,6 +103,12 @@ var/runechat_icon = null
 	if (!ismob(target))
 		extra_classes |= "small"
 
+	// If we heard our name, it's important
+	var/first_word = get_first_word(owner.name)
+	var/last_word = get_last_word(owner.name)
+	text = replacetext_char(text, first_word, "<b>[first_word]</b>")
+	text = replacetext_char(text, last_word, "<b>[last_word]</b>")
```

Comment:
I'm pretty sure there's no point in using the `_char` version if you're not using the position args